cpp/src/reductions/scan.cu

```diff
@@ -116,29 +120,36 @@ namespace { //anonymous
         }
 
         static
-            gdf_error exclusive_sum(void *&temp_storage, size_t &temp_storage_bytes,
-                const T *input, T *output, size_t size, cudaStream_t stream) {
-            cub::DeviceScan::ExclusiveSum(temp_storage, temp_storage_bytes,
-                input, output, size, stream);
+        gdf_error exclusive_scan(void *&temp_storage, size_t &temp_storage_bytes,
+            const T *input, T *output, size_t size, cudaStream_t stream)
+        {
+            Op op;
```

Comment:
Minor simplification. You can just pass a temporary `Op{}` into `cub::DeviceScan::ExclusiveScan` instead of explicitly constructing an `l-value` here. 

Likewise is true of the `identity`.